### PR TITLE
KevS-SK1552-Update-Glue-records-for-DomainDNS

### PR DESCRIPTION
**Pull request by Kazarus4**
Updates : 

- Date of last update
- Add an alert paragraph into the step 2 to explain the differences between object and attributes for eu, it, de and be tld.
- Add an alert paragraph into the step 4 to precise to customer that it must be add the IP associated to his DNS server personalized. this to be run correctly.

Need peer rv by @OVH-Team-Guides before tech rv, FRCA duplication and translations for the other subs.

### CODE REVIEW

File: pages/web_cloud/domains/glue_registry/guide.fr-fr.md [hunk 1]
```diff
@@ -58,6 +58,13 @@ Si besoin et pour plus d'informations, consultez notre tutoriel sur l'outil [Zon
 
 ### Étape 2 : ajouter les enregistrements « GLUE » <a name="step2"></a>
 
+> [!alert]
+>
+> Pour les extensions *.eu*,*.it*, *.be* et *.de*, les registres de ces extensions ne considèrenet pas les enregistrements « GLUE » comme des « objets » mais comme des « attributs ».
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/glue_registry/guide.fr-fr.md]:
```suggestion
> Les registres des extensions *.eu*,*.it*, *.be* et *.de* ne considèrent pas les enregistrements « GLUE » comme des « objets » mais comme des « attributs ».
```

File: pages/web_cloud/domains/glue_registry/guide.fr-fr.md [hunk 2]
```diff
@@ -58,6 +58,13 @@ Si besoin et pour plus d'informations, consultez notre tutoriel sur l'outil [Zon
...
+>
+> Par conséquent et pour ces extensions, passez **directement à l'étape 3** du présent guide sans réaliser l'étape 2.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/glue_registry/guide.fr-fr.md]:
```suggestion
> Par conséquent, pour ces extensions, passez **directement à l'étape 3** de ce guide sans réaliser l'étape 2.
```

File: pages/web_cloud/domains/glue_registry/guide.fr-fr.md [hunk 3]
```diff
@@ -130,6 +137,13 @@ Pour cela, connectez-vous à votre [espace client OVHcloud](/links/manager){.ext
  
 Positionnez-vous sur l'onglet `Serveurs DNS`{.action} puis cliquez sur `Modifier les serveurs DNS`{.action}. Remplacez alors vos serveurs DNS actuels par ceux que vous souhaitez utiliser en tant que serveur DNS personnalisés. 
 
+> [!alert]
+>
+> Si vos serveurs DNS personnalisées ont été créées avec les extensions *.eu*,*.it*, *.be* et/ou *.de*, renseignez **obligatoirement** l'adresse IP asociée respectivement à chacun de vos serveurs DNS personnalisés.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/glue_registry/guide.fr-fr.md]:
```suggestion
> Si vos serveurs DNS personnalisés ont été créés avec les extensions *.eu*,*.it*, *.be* ou *.de*, renseignez **obligatoirement** l'adresse IP associée respectivement pour chacun de vos serveurs DNS personnalisés.
```

File: pages/web_cloud/domains/glue_registry/guide.fr-fr.md [hunk 4]
```diff
@@ -130,6 +137,13 @@ Pour cela, connectez-vous à votre [espace client OVHcloud](/links/manager){.ext
...
+>
+> Le cas échéant, les serveurs DNS personnalisés ne seront pas pris en compte correctmeent et ne fonctionneront donc pas avec votre nom de domaine.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/glue_registry/guide.fr-fr.md]:
```suggestion
> Sans cela, les serveurs DNS personnalisés ne seront pas pris en compte correctement et ne fonctionneront donc pas avec votre nom de domaine.
```

**Review by OVH-Team-Guides — COMMENTED** [review]
alert! is a bit too stong here IMO

File: pages/web_cloud/domains/glue_registry/guide.fr-fr.md [hunk 5]
```diff
@@ -58,6 +58,13 @@ Si besoin et pour plus d'informations, consultez notre tutoriel sur l'outil [Zon
 
 ### Étape 2 : ajouter les enregistrements « GLUE » <a name="step2"></a>
 
+> [!alert]
```
Comment (Kazarus4) [inline pages/web_cloud/domains/glue_registry/guide.fr-fr.md]:
```suggestion
> [!warning]
```

File: pages/web_cloud/domains/glue_registry/guide.fr-fr.md [hunk 6]
```diff
@@ -130,6 +137,13 @@ Pour cela, connectez-vous à votre [espace client OVHcloud](/links/manager){.ext
  
 Positionnez-vous sur l'onglet `Serveurs DNS`{.action} puis cliquez sur `Modifier les serveurs DNS`{.action}. Remplacez alors vos serveurs DNS actuels par ceux que vous souhaitez utiliser en tant que serveur DNS personnalisés. 
 
+> [!alert]
```
Comment (Kazarus4) [inline pages/web_cloud/domains/glue_registry/guide.fr-fr.md]:
```suggestion
> [!warning]
```

**Review by OVH-Team-Guides — APPROVED** [review]

**Pull request merged**